draft-shafranovich-rfc4180-bis.md

```diff
@@ -150,31 +160,43 @@ of {{?RFC7231}}). However, some implementations MAY use other values.
 The ABNF grammar (as per {{!RFC5234}}) appears as follows:
 
 ~~~~~~~~~~
-file = *(record linebreak)
+file = *((record / comment) linebreak)
 
 record = field *(COMMA field)
 
+comment = HASH *COMMENTDATA
```

Comment:
Why not use TEXTDATA instead COMMENTDATA?

Reply:
Because TEXTDATA does not allow COMMA and DQUOTE – which is totally fine in comments.